readium/navigator/build.gradle.kts

```diff
@@ -70,7 +71,7 @@ dependencies {
     implementation(libs.androidx.viewpager2)
     implementation(libs.androidx.webkit)
     // Needed to avoid a crash with API 31, see https://stackoverflow.com/a/69152986/1474476
-    implementation("androidx.work:work-runtime-ktx:2.7.1")
+    implementation("androidx.work:work-runtime-ktx:2.8.1")
```

Comment:
After a quick test on 31 emulator, I think this is not needed anymore. At least with the media3 navigators, we will deprecate the media1 and media2 ones.